.github/workflows/_extension_distribution.yml

```
@@ -658,7 +658,7 @@ jobs:
 
       - uses: mymindstorm/setup-emsdk@v13
         with:
-          version: 'latest'
+          version: 3.1.74
```

```suggestion
          version: 3.1.71
```
3.1.71 is currently the version used to build duckdb-wasm, I would lock to that also for extensions.